### PR TITLE
Initial design

### PR DESCRIPTION
**Pull request by Maxgrb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,19 @@ on the browser.
 
 ## How to use this repo
 
-* Clone it, and `cd` into it
-* `npm install`
-* `npm start`
+1. Clone it, and `cd` into it
+2. Install [Node.js](https://nodejs.org/en/).
+3. Install Node Package Manager `sudo npm install npm -g`.
```
Comment (wholesomedev) [inline README.md]: I think we should remove points 2 and 3 because anyone who works on this repo should already be familiar with and have Node and npm.

**Review by wholesomedev — DISMISSED** [review]
Thanks for this work @Maxgrb!

I think the code looks good. I just have a few requests about the repo structure in general.

This repo is intended to be for dev mode only. There should not be a production mode for it. Therefore, we should not have the following:

* `npm run build` (`npm run start` should be enough)
* separate folders for `public/` and `build/`
* separate instructions for Build and Develop in the `README.md`

Other than that, I think the work is excellent.

**Comment by wholesomedev** [comment]
@Maxgrb So based on my earlier review, I've made [these changes](https://github.com/hollowverse/design/pull/2/commits/f777a5b84a74df95cd4743ed148558daf3dba8ea) and pushed a commit to this PR. Let me know what you think.

The other issue that I just noticed is that the text color in here is white against a yellow background. The contrast between the colors is too low

image

Another thing regarding the font in general is that it is not very legible. It looks okay on a retina display but it doesn't look good on my 1080p computer monitor. Also, I'm not sure if fancy fonts are worth the additional load time. I would like us to stick to the legible traditional fonts if possible. What do you think?

@forabi please feel free chime in on this. Give this PR a look and let us know if you have any thoughts. Just note that the main concern in `hollowverse/design` is the design and related issues.

**Comment by Maxgrb** [comment]
@msafih thanks for feedback and commit. Dev mode of design repo makes sense.

I've made changes.

Also I've removed font smoothing. It may make fonts better on non-retina display.

About fonts in general. We could use system fonts. But they are mostly different between platforms. The website will be look differently on different platforms. There are "safe" fonts ([http://www.ampsoft.net/webdesign-l/WindowsMacFonts.html](http://www.ampsoft.net/webdesign-l/WindowsMacFonts.html)), but they are pretty plain. I think, using system fonts is OK, if branding isn't priority.

We could test different typefaces using [https://fonts.google.com](https://fonts.google.com/). They require additional loading time, but it'll make typography and brand consistent across platforms.

We can request typefaces directly from Google. There is a chance depending on font popularity, that they was cached and won't be require additional loading time.

**Comment by forabi** [comment]
@Maxgrb Great work Max! I just noticed a problem with the name of the notable person. If the name is too long, it wraps below the image, showing too much white space and making it hard to read.

It looks like this happens because the image has `float: left;`. Since we are only targeting modern browsers, we can safely use Flexbox instead of floats. This will probably make it easier to position the text correctly.

@msafih Now that I think about it, I am not sure the current layout for the image and the title is good for long names. The right side area is just too small on mobile screens. Even if we could fit the entire name on the right, it would probably break in the middle of the first or last name.

localhost-8080- iphone 6

**Comment by wholesomedev** [comment]
Good points @forabi. I'm thinking let's merge this PR as-is and open new issues for @Maxgrb to work on. One issue will be migrating the design to flexbox and the second issue is coming up with a different layout that will leave enough space for the NPs name on mobile.

**Review by wholesomedev — APPROVED** [review]

**Pull request merged**